### PR TITLE
Ensure all writer endpoints set info on span

### PR DESCRIPTION
**Pull request by jotare**

### CODE REVIEW

**Review by bloodbare — CHANGES_REQUESTED** [review]
needs to be conditional that tracer provider exist

File: nucliadb/nucliadb/writer/api/v1/upload.py [hunk 1]
```diff
@@ -354,6 +354,13 @@ async def patch(
     if rslug:
         rid = await get_rid_from_params_or_raise_error(kbid, slug=rslug)
 
+    if rid:
```
[the remark this answers is not in the document]
Comment (jotare) [reply nucliadb/nucliadb/writer/api/v1/upload.py:357]: But only if `rslug` is passed. You can call the endpoint only passing `upload_id`

[the remark this answers is not in the document]
Comment (lferran) [reply nucliadb/nucliadb/writer/api/v1/upload.py:357]: ah totally right!

**Review by bloodbare — APPROVED** [review]
rereviewed

**Review by bloodbare — APPROVED** [review]

**Pull request merged**